mat64/cholesky.go

```diff
@@ -178,3 +178,155 @@ func (s *SymDense) InverseCholesky(chol *Cholesky) error {
 	s.SymOuterK(1, &t)
 	return err
 }
+
+// RankOne performs a rank-1 update (or downdate when alpha is negative) of the
```

Comment:
"Downdate/ing" sounds strange but after having studied the related literature for the past two weeks "downdate" is commonly used in this field to distinguish doing A - v v' from doing A + v v'. I will remove it here, splashing the word in the first line of the comment is not good.